libs/utils/utils.js

```diff
@@ -511,6 +515,30 @@ export function isInTextNode(node) {
   return (node.parentElement.childNodes.length > 1 && node.parentElement.firstChild.tagName === 'A') || node.parentElement.firstChild.nodeType === Node.TEXT_NODE;
 }
 
+function lingoActive() {
+  const langFirst = (getMetadata('langFirst') || PAGE_URL.searchParams.get('langFirst'))?.toLowerCase();
+  return ['true', 'on'].includes(langFirst);
+}
+
+/**
+ * Get user's country code for mep-lingo region detection.
+ * TODO: Finalize source of truth with Victor/Vivian. Current priority:
+ *   1. akamaiLocale URL param (for testing/override)
+ *   2. akamai sessionStorage
+ *   3. Server timing geo header
+ * Proposed priority (needs discussion):
+ *   1. param override
+ *   2. cookie (TBD if using cookie)
+ *   3. server info (with region object lookup)
+ */
+export function getUserCountry() {
+  const urlParams = new URLSearchParams(window.location.search);
+  return urlParams.get('akamaiLocale')?.toLowerCase()
+    || sessionStorage.getItem('akamai')
+    || window.performance?.getEntriesByType('navigation')?.[0]?.serverTiming
+      ?.find((t) => t?.name === 'geo')?.description?.toLowerCase();
+}
```

Comment:
This is quite hard to read. It would probably be preferable, especially since it is in utils, to store these return values as semantic variables and then return chain
```
const akamaiParams = urlParams.get('akamaiLocale')?.toLowerCase();
const akamaiStored = sessionStorage.getItem('akamai');
const performanceGeo = window.performance?.getEntriesByType('navigation')?.[0]?.serverTiming
      ?.find((t) => t?.name === 'geo')?.description?.toLowerCase();
return akamaiParams || akamaiStored || performanceGeo
```